app/templates/login.html

```diff
@@ -0,0 +1,24 @@
+{% extends "index.html" %}
+
+{% block body %}
+
+<div class="container">
+    <div>
+        <form method="POST" action="/login">
+            <div class="form-group">
+                <label for="exampleInputUsername"><kbd>Your User Name</kbd></label>
+                <input type="text" class="form-control" value="" name="user_name" placeholder="@user-name">
+            </div>
+            <div class="form-group">
+                <label for="exampleInputPassword1"><kbd>Password</kbd></label>
+                <input type="password" class="form-control" valuye="" name="password" placeholder="*********">
+            </div>
+            <input type="submit" value="Submit" class="btn btn-default">
+            <button class="btn btn-default"><a href="register">Forgot Password?</a></button>
+        </form>
+    </div>
+    <br>
+    <p><kbd>New Here? <a href="register">Sign Up Here</a></kbd></p>           
```

Comment:
Does the href="register" work? Try pointing it to a link using url_for(link_route)